### PR TITLE
feat: frontend (bc-apis & bc-console )as a component

### LINKED ISSUE #25: frontend (bc-apis & bc-console )as a component

### PR DESCRIPTION
**Pull request by dayuy**
Fix #25

### CODE REVIEW

**Comment by bjwswang** [comment]
> Fix #25

@dayuy  Thanks! This is great! Could you pls 

- add a README like https://github.com/kubebb/components/tree/main/charts/kubebb-core
- add a componentplan like https://github.com/kubebb/components/tree/main/examples/minio

**Review by 0xff-dev — APPROVED** [review]

**Review by Abirdcfly — APPROVED** [review]

File: charts/bc-apis/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+```shell
+    helm delete bc-apis
+```
+
+## Configuration
+
+| Parameter                    | Description | Default                                       |
+|------------------------------|-------------|-----------------------------------------------|
+| `ingressDomain`              | B           |                                               |
```
Comment (dayuy) [inline charts/bc-apis/README.md:31]: Description needs to be supplemented.

Comment (bjwswang) [reply charts/bc-apis/README.md:31]: maybe `Domain of the ingress controller that exposes the application to the outside world`

**Comment by bjwswang** [comment]
@dayuy bump `bc-depository` `bc-explorer` to `v0.0.2`

**Comment by dayuy** [comment]
```shell
helm install bc-apis -n baas-system ./bc-apis-v0.0.1.tgz
```

**Review by bjwswang — APPROVED** [review]

**Pull request merged**